x-pack/legacy/plugins/infra/public/components/logging/log_minimap/log_minimap.tsx

```diff
@@ -152,7 +245,18 @@ const TimeCursor = euiStyled.line`
       : props.theme.eui.euiColorDarkShade};
 `;
 
-const MinimapWrapper = euiStyled.svg`
+const MinimapWrapper = euiStyled.svg<any>`
```

Comment:
Trying to figure out a better way to get this to pass typechecking without resorting to `<any>`

Reply:
As it turns out that's just how `<>` works in TypeScript lol